conf/messages

```diff
@@ -2069,13 +2069,13 @@ upload-supporting-evidence.check-your-answers.assistive=the file named {0}
 upload-supporting-evidence.check-your-answers.cya-delete=Delete
 upload-supporting-evidence.check-your-answers.add-another-file=Add another file
 
-upload-supporting-evidence.expired.title=Your uploaded supporting evidence has expired
-upload-supporting-evidence.expired.p1=Uploaded files can only be kept for 7 days before they expire. You can either upload these files again or delete them from your return.
+upload-supporting-evidence.expired.title=For your security, we deleted these documents
+upload-supporting-evidence.expired.p1=We save documents for 7 days.
```

Comment:
`We save documents for 7 days` might be better worded as `We save documents for 7 days only`

Reply:
this came from Jean so I won't change it unless we really want to bring it up with her